astropy/modeling/utils.py

```diff
@@ -484,3 +484,64 @@ def combine_labels(left, right):
 
     return left + right
 
+
+def render_models2sky(model_list, image, x_key='x_mean', y_key='y_mean'):
+    """
+    Evaluates a list of 2D models on an 2D image array. Limits the evaluation to the
```

Comment:
Could you extend/rewrite the docstring a bit according to the standard convention?